templates/Copy DICOM Metadata Changes to ADLS Gen2 in Delta Format/manifest.json

```diff
@@ -0,0 +1,44 @@
0004 C 9.12301,0.9109999999999996 9.16001,1.1600000000000001 9.03001,1.3369999999999997 L 4.161999999999999,7.57 C 3.9730000000000003,7.811999999999999 3.6119999999999997,7.827999999999999 3.4019999999999997,7.603 L 0.1080000000000001,4.0649999999999995 z\" transform=\"matrix(1, 0, 0, 1, 1.4849949999999996, 2.1985)\"/></g></g><g transform=\"matrix(1, 0, 0, 1, 0, 84)\"><g transform=\"matrix(1, 0, 0, 1, 0, 30)\"><path stroke=\"none\" fill=\"#ffffff\" d=\"M 0,0 L 12,0 L 12,12 L 0,12 z\" transform=\"matrix(1, 0, 0, 1, 0, 0)\" filter=\"drop-shadow(0px 0px 1px #0078dA)\"/><path stroke=\"none\" fill=\"#A80000\" d=\"M 9.38802,1.4220000000000002 L 7.9660199999999985,0 L 4.693999999999999,3.272 L 1.4220000000000002,0 L 0,1.4220000000000002 L 3.272,4.694000000000001 L 0,7.96597 L 1.4220000000000002,9.387970000000001 L 4.693999999999999,6.116 L 7.9660199999999985,9.387970000000001 L 9.38802,7.96597 L 6.116,4.694000000000001 L 9.38802,1.4220000000000002 z\" transform=\"matrix(1, 0, 0, 1, 1.3059900000000004, 1.3060149999999995)\"/></g></g><g transform=\"matrix(1, 0, 0, 1, 0, 126)\"><g transform=\"matrix(1, 0, 0, 1, 0, 30)\"><path stroke=\"none\" fill=\"#ffffff\" d=\"M 0,0 L 12,0 L 12,12 L 0,12 z\" transform=\"matrix(1, 0, 0, 1, 0, 0)\" filter=\"drop-shadow(0px 0px 1px #0078dA)\"/><path stroke=\"none\" fill=\"#0F6CBD\" d=\"M 0,3.78033 C 0,3.3661200000000004 0.33579000000000003,3.03033 0.75,3.03033 L 6.68934,3.03033 L 4.71967,1.0606600000000004 C 4.42678,0.7677700000000001 4.42678,0.2928900000000003 4.71967,0 C 5.01256,-0.2928899999999999 5.48744,-0.2928899999999999 5.78033,0 L 9.0303,3.25 C 9.3232,3.54289 9.3232,4.01775 9.0303,4.31064 L 5.78035,7.56084 C 5.48746,7.85373 5.01259,7.85373 4.71969,7.56087 C 4.42678,7.26799 4.42677,6.793120000000001 4.71965,6.50021 L 6.68942,4.53033 L 0.75,4.53033 C 0.33579000000000003,4.53033 0,4.19454 0,3.78033 z\" transform=\"matrix(1, 0, 0, 1, 1.4848499999999998, 2.219565)\"/></g></g></g><g class=\"gojs-ts\"/></g></g><g transform=\"matrix(0.303951367781155, 0, 0, 0.303951367781155, 129.48328267477203, 90.63829787234043)\"/></g><clipPath id=\"mainClip614222\"><rect x=\"0\" y=\"0\" width=\"400px\" height=\"240px\"/></clipPath><linearGradient x1=\"222.68297205000002\" x2=\"222.68297205000002\" y1=\"172.60750124999998\" y2=\"0\" id=\"GRAD614223\" gradientUnits=\"userSpaceOnUse\"><stop offset=\"0\" stop-color=\"#32BEDD\"/><stop offset=\"0.532\" stop-color=\"#50E6FF\"/><stop offset=\"1\" stop-color=\"#9CEBFF\"/></linearGradient><linearGradient x1=\"124.9601562\" x2=\"124.9601562\" y1=\"189.83581249999997\" y2=\"56.663879375\" id=\"GRAD614224\" gradientUnits=\"userSpaceOnUse\"><stop offset=\"0\" stop-color=\"#0078D4\"/><stop offset=\"1\" stop-color=\"#5EA0EF\"/></linearGradient><linearGradient x1=\"124.9601562\" x2=\"124.9601562\" y1=\"189.83581249999997\" y2=\"56.663879375\" id=\"GRAD614225\" gradientUnits=\"userSpaceOnUse\"><stop offset=\"0\" stop-color=\"#0078D4\"/><stop offset=\"1\" stop-color=\"#5EA0EF\"/></linearGradient></svg>",
```

Comment:
The image still isn't generating properly from ADF (both the stable and preview portals) and Synapse.